Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -654,18 +549,6 @@ hr {
     background-color: $primary;
 }
 
-.pop-over-close {
-    float: right;
-    padding-top: 0px;
-    color: black;
-    border: none;
-    background-color: transparent;
-}
-
-.popover-content {
-    max-width: 20rem;
-    color: black;
-}
```

Comment:
Are these classes not used by the other popover (export records)?

Reply:
I replaced the parts that were needed on the other popover with bootstrap classes like 'float-right' and 'text-dark'.